packages/site-kit/src/lib/styles/tokens.css

```diff
@@ -41,8 +44,11 @@
 		--sk-back-5: hsl(0, 0%, 40%);
 		--sk-back-translucent: hsla(0, 0%, 100%, 0.1);
 		--sk-theme-1: hsl(15, 100%, 55%);
+		--sk-theme-1-hsl: 15, 100%, 55%;
```

Comment:
Is it possible to deduplicate this by doing `--sk-theme-1: hsl (var(--sk-theme-1-hsl))`?

Reply:
Good catch!